### DISCUSSION TITLE
Cisco IOS output to json

### DISCUSSION THREAD

**Discussion by akshayavk96** [ktbyers/netmiko#2873] [Q&A]
Hello,
Is there a way to convert the output of Netmiko send command (cisco output)  to JSON or XML?
I tried setting , use_textfsm=true, but it doesn't seem to be working. Any help would really be appreciated. Thanks in advance!

**Comment by ktbyers** [comment]
@akshayavk96 Which particular `show command` are you looking for?

The main options are using Textfsm or Cisco's genie, but these options will only work for specific show commands. They will not work for all show commands. 

These would return Python data structures which you would then need to convert to JSON.

**Reply by akshayavk96** [reply]
Thank you for the response. I a looking at grabbing all the ssl certificates associated with a device. Particularly used the " show certificate all" command.  I have tried using textfsm but it didn't work for that command.
